.travis.yml

```diff
@@ -53,7 +53,10 @@ matrix:
       before_script: cd integration && ./travis/prepare_integration.sh
       script: ./travis/run_integration.sh --pools=off --auth=http-basic
 
-    - name: 'Cook Scheduler integration tests with job launch rate limit'
+# We want a small rate limit to make the job launch rate limit unit test be stable and not
```

Comment:
Here we call it a unit test, but on the next line we refer to integration tests

Reply:
Thanks. Fixed.